backend.native/cli.bc/src/org/jetbrains/kotlin/cli/bc/K2Native.kt

```diff
@@ -153,6 +153,7 @@ class K2Native : CLICompiler<K2NativeCompilerArguments>() {
                 put(LIGHT_DEBUG, arguments.lightDebug)
                 put(STATIC_FRAMEWORK, selectFrameworkType(configuration, arguments, outputKind))
                 put(OVERRIDE_CLANG_OPTIONS, arguments.clangOptions.toNonNullList())
+                put(ALLOCATION_MODE, arguments.allocationMode)
```

Comment:
"Allocation mode" is somewhat unclear term. Maybe use more common "allocator" instead?